pom.xml

```diff
@@ -570,6 +570,12 @@
             </dependency>
 
             <dependency>
+                <groupId>joda-time</groupId>
+                <artifactId>joda-time</artifactId>
+                <version>2.9.9</version>
```

Comment:
Add `        <dep.joda.version>2.9.9</dep.joda.version>` to `<properties>` section instead.